radix-engine/src/object_modules/metadata/package.rs

```diff
@@ -16,10 +16,9 @@ use super::{RemoveMetadataEvent, SetMetadataEvent};
 
 #[derive(Debug, Clone, Eq, PartialEq, ScryptoSbor)]
 pub enum MetadataError {
-    KeyStringExceedsMaxLength { max: usize, actual: usize },
```

Comment:
We can't remove these error variants as it breaks the error deserialization of historic receipts in the node (until we fix that to store historic schemas - which I really want to do one day soon ;))

Even if they're unused, we need to keep them for now I'm afraid.

Reply:
You're right. Thought we had that error to string conversion already.

Reply:
We do have error to string conversion, but to save space, it happens on the API at response serving time; and uses the _current_ `RuntimeError`. I'd like to store some historic schemas at each protocol update in the node so we can convert with these old schemas whilst still updating the errors.